nuxt.config.ts

```diff
@@ -25,4 +25,9 @@ export default defineNuxtConfig({
   typescript: {
     typeCheck: true,
   },
+  runtimeConfig: {
+    public: {
+      apiUrl: process.env.SNAP_API_URL,
```

Comment:
Suggest adding default if unspecified, i.e. `|| 'https://earthmaps.io'

Reply:
Whoops, looks like I forgot to commit the `.env` file that had the default for `SNAP_API_URL`. But I'll go ahead and take the approach suggested here since that's how our other Nuxt apps do it.

Reply:
I've added a default apiUrl (`|| 'https://earthmaps.io/`) to `nuxt.config.ts` as described above in d1f661f. We discovered that `.env` is currently ignored via `.gitignore`, so `nuxt.config.ts` is where the default setting belongs.